<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stu_manage</class>
 <widget class="QMainWindow" name="stu_manage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AtQlas Alpha</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image:url(:/atqres_ico/flat_stu.png)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolBox" name="drawer">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>791</width>
       <height>271</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>238</red>
           <green>238</green>
           <blue>238</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ToolTipText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>238</red>
           <green>238</green>
           <blue>238</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ToolTipText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>127</red>
           <green>127</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>127</red>
           <green>127</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>148</red>
           <green>148</green>
           <blue>148</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ToolTipText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>205</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent</string>
      </property>
      <attribute name="label">
       <string>My Current Lessons</string>
      </attribute>
      <widget class="QTableWidget" name="current_lesson_table">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>721</width>
         <height>201</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>PT Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget 
{background-color: transparent;}
QScrollBar:vertical {
    background: url(images/pure_lite_gray.png);
    width: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:vertical {
    background: rgb(195, 195, 195);
    min-height: 30px;
    margin: 0 1px 0 2px;
    border-radius: 3px;
    border: none;
}

QScrollBar:horizontal {
    height: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:horizontal {
    background: rgb(195, 195, 195);
    min-width: 20px;
    margin: 2px 0px 1px 0px;
    border-radius: 3px;
    border: none;
}
QHeaderView::section
{
border-image:url(:/atqres_ico/pure_lite_gray.png);
height:30px;
spacing: 20px;
background-color:rgb(39,49,58);
color: white;

text-align: right;
font-family: Courier;
font-size:12px;
}</string>
       </property>
      </widget>
      <widget class="QPushButton" name="delete_course_button">
       <property name="geometry">
        <rect>
         <x>740</x>
         <y>30</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
border-image:url(:/atqres_ico/lite_wrong.png)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="delete_course_button_2">
       <property name="geometry">
        <rect>
         <x>736</x>
         <y>100</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
border-image:url(:/atqres_ico/refresh.png)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>205</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent</string>
      </property>
      <attribute name="label">
       <string>Add New Lessons</string>
      </attribute>
      <widget class="QTableWidget" name="choose_lesson_table">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>721</width>
         <height>191</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>PT Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget 
{background-color: transparent;}
QScrollBar:vertical {
    background: url(images/pure_lite_gray.png);
    width: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:vertical {
    background: rgb(195, 195, 195);
    min-height: 30px;
    margin: 0 1px 0 2px;
    border-radius: 3px;
    border: none;
}

QScrollBar:horizontal {
    height: 0px;
    margin: 0px 0 0px 0;
}
QScrollBar::handle:horizontal {
    background: rgb(195, 195, 195);
    min-width: 20px;
    margin: 2px 0px 1px 0px;
    border-radius: 3px;
    border: none;
}
QHeaderView::section
{
border-image:url(:/atqres_ico/pure_lite_gray.png);
height:30px;
spacing: 20px;
background-color:rgb(39,49,58);
color: white;

text-align: right;
font-family: Courier;
font-size:12px;
}</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>70</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
border-image:url(:/atqres_ico/lite_plus_all.png)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>120</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
border-image:url(:/atqres_ico/lite_plus.png)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="sid_label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>158</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Student ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="klass_label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>158</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>class</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>158</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
    <widget class="QLabel" name="grade_label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>158</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>grade</string>
     </property>
    </widget>
    <widget class="QLabel" name="school_label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>130</y>
       <width>158</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>school</string>
     </property>
    </widget>
    <widget class="QLabel" name="gpa_label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>210</y>
       <width>158</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>GPA</string>
     </property>
    </widget>
    <widget class="QLabel" name="sid_label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>158</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Student ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>158</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
    <widget class="QLabel" name="klass_label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>158</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>class</string>
     </property>
    </widget>
    <widget class="QLabel" name="school_label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>158</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>school</string>
     </property>
    </widget>
    <widget class="QLabel" name="grade_label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>158</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>grade</string>
     </property>
    </widget>
    <widget class="QLabel" name="gpa_label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>158</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>GPA</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>200</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>PT Mono</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/pure_gray_7.png)</string>
     </property>
     <property name="placeholderText">
      <string>Type here to filter...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border-image:url(:/atqres_ico/cock_back.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
